core/src/main/scala/org/apache/spark/deploy/master/ui/MasterPage.scala

```diff
@@ -73,6 +73,14 @@ private[spark] class MasterPage(parent: MasterWebUI) extends WebUIPage("") {
           <div class="span12">
             <ul class="unstyled">
               <li><strong>URL:</strong> {state.uri}</li>
+              {
+                state.restUri.map { uri =>
+                  <li>
+                    <strong>REST URL:</strong> {uri}
+                    <span class="rest-uri"> (for standalone cluster mode in Spark 1.3+)</span>
```

Comment:
I think it's fine to shorten this to say just say "(cluster mode)" only. The fact that the cluster is a standalone mode cluster and that it is running Spark 1.3 is self-evident.